lepotato/overlays/boot/boot.cmd

```diff
@@ -0,0 +1,70 @@
+# DO NOT EDIT THIS FILE
+#
+# Please edit /boot/armbianEnv.txt to set supported parameters
```

Comment:
Does the lepotato not use the default distro boot? If it does better to install `u-boot-menu` instead of a big handcoded script